CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `libcnb-cargo`:
   - No longer outputs paths for non-libcnb.rs and non-meta buildpacks. ([#657](https://github.com/heroku/libcnb.rs/pull/657))
   - Build output for humans changed slightly, output intended for machines/scripting didn't change. ([#657](https://github.com/heroku/libcnb.rs/pull/657))
+- `libcnb-package`
```

Comment:
As of #662 we're treating `libcnb-package` as an internal crate and so not including changes to it in the changelog, but instead only mentioning things that affect end-users of `libcnb-cargo`.

So this changelog entry should perhaps (a) go under `libcnb-cargo` not `libcnb-package`, (b) only mention that `.gitignore` is now respected, and not mention the signature change of `find_buildpack_dirs`at all.